dev/NavigationView/NavigationView_rs1_themeresources.xaml

```diff
@@ -267,6 +270,7 @@
     <Thickness x:Key="TopNavigationViewOverflowMenuPadding">0,8</Thickness>
     <Thickness x:Key="NavigationViewItemChildrenMenuFlyoutPadding">0,8</Thickness>
 
+    <x:Double x:Key="NavigationViewItemExpandedGlyphFontSize">12.0</x:Double>
```

Comment:
I am using this resource in both Left and Top NavView modes. Is there a case to make to create a copy of this resource for Top navigation (TopNavigationViewItemExpandedGlyphFontSize)? I *think* we should be fine with just one resource here for both modes but perhaps there is a valid case to make here.

Same goes for the flyouts (overflow menu or children menu in LeftCompact/LeftMinimal/Top modes). Does it make sense here to create FontSize specific theme resources for those UI elements? I.e. could someone might want to use a smaller chevron in the flyout menu than for the chevron used in top-level items?

Thoughts about this?

Reply:
I think that with just one resource it would still be possible to achieve this effect, by updating the resource at the same time you change your navigation view from left to top mode.  Undating resources from code is a bit annoying though, adding a second one would make this easier, however the situation seems very rare. @YuliKl can you weigh in?

Reply:
In Top mode, for example, there is the expand/collapse chevron for top-level items and then there are the children in the flyouts, which might also have have children, so show an expand/collapse chevron as well. With just one resource, developers couldn't change the font size of the chevron depending on whether it belongs to a top-level item or to a child item displayed in the flyout.

The resource introduced for these flyout children chevrons in Top mode could be named `TopNavigationViewItemOnOverflowExpandedGlyphFontSize`.

Reply:
I would prefer to keep resources as streamlined as possible and only add one. Personally, I prefer visual consistency between the chevron sizes. More importantly, I would like to wait for an app developer to request this differentiated scenario instead of future-proofing for something that may not be necessary.

Reply:
@YuliKl Sounds good to me. The reason why I am asking is that if we now decide to use one resource in all places, then if in the future we indeed do want to add finer customization options, we might end up with "UI breaking changes" (because the customized resource would no longer be applied in all cases it would be today). This would be resolved with proposal https://github.com/microsoft/microsoft-ui-xaml/issues/2913 though and there seems to be interest on the team to make it happen so I guess we will be fine here with just one resource for now.